src/features/movies/moviesSlice.js

```diff
@@ -0,0 +1,41 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+const moviesSlice = createSlice({
+  name: "movies",
+  initialState: {
+    movies: [],
+    genres: [],
+    loadingStatus: "loading",
```

Comment:
I think we're not managing just loading state. It can be success, error or loading so maybe just call it status or fetchingStatus

Reply:
Alright, I'll have to do the same changes in peopleSlice.js